stylesheets/stylesheet.css

```diff
@@ -0,0 +1,102 @@
+* {
+    font-size: 62.5%;
```

Comment:
Putting your font-size here means the % multiplier will stack with nested elements, making fonts inside a few divs smaller than expected even if you use rems - put it in a root tag like \<html> OR \<body>